resources/executables/windows/language-server-launcher.bat

```diff
@@ -0,0 +1,131 @@
+@echo off
+
+REM ---------------------------------------------------------------------------
+REM   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@rasika  Shall we change year to 2019?

Reply:
@tharik these scripts are directly copied from https://github.com/ballerina-platform/ballerina-lang/commit/822cb90ae2be1df2775923eeea1517c3da47777b